images/win/scripts/Installers/Install-Vcpkg.ps1

```diff
@@ -3,21 +3,23 @@
 ##  Desc:  Install vcpkg
 ################################################################################
 
-Import-Module -Name ImageHelpers -Force
+Write-Host "Download the latest vcpkg version..."
+$vcpkgDownloadUrl = (Invoke-RestMethod "https://api.github.com/repos/microsoft/vcpkg/releases/latest").tarball_url
```

Comment:
maybe zipball? And then use `Extract-7Zip` function instead of tar since I'm not so confident in Windows tar

Reply:
We can't unzip archive without first directory with `Extract-7Zip`.  That's why i'm using `tar`.